README.md

```diff
@@ -1,2 +1,5 @@
 # serverless-api
 AWS: API, Dynamo and Lambda - 401
+
+
+![Whiteboard](/assets/uml.png)
```

Comment:
I would like to see some information added to your README:
- What inputs are required
- What outputs are required
- What is the root URL to your API

I would also like to see an attempt at tests